test/unit/spec/IDPDiscovery_spec.js

```diff
@@ -594,6 +594,21 @@ function (Q, OktaAuth, WidgetUtil, Okta, Util, AuthContainer, IDPDiscoveryForm,
     });
 
     Expect.describe('Device Fingerprint', function () {
+      itp(`does not compute device fingerprint if security image is off, deviceFingerprinting
+        is true and useDeviceFingerprintForSecurityImage is true)`, function () {
```

Comment:
seems that there is an extra parenthesis in the description... maybe we can shorten this to `is not computed if securityImage is off, deviceFingerprinting is true and useDeviceFingerprintForSecurityImage is true`?